first_order_model/shrink_util.py

```diff
@@ -1128,11 +1124,15 @@ def try_reduce(curr_loss, curr_model, dataloader, layer_graph,
     generator_full = GeneratorFullModel(new_kp_detector, model_copy, new_discriminator, train_params)
     discriminator_full = DiscriminatorFullModel(new_kp_detector, model_copy, new_discriminator, train_params)
     counter = 0
-    for k in dataloader:
-        break
     generator_full = DataParallelWithCallback(generator_full, device_ids=device_ids)
     discriminator_full = DataParallelWithCallback(discriminator_full, device_ids=device_ids)
 
+    total_loss = get_metrics_loss(metrics_dataloader, lr_size, generator_full,
+                                  generator_type)
+    print("Loss for this model is: ", total_loss)
+    for x in dataloader:
```

Comment:
please do not use these things -- they're doing nothing useful

Reply:
The loss is also just generally useful. I can comment it out

Reply:
I meant the break after dataloader

Reply:
Done